src/main/java/com/impactupgrade/nucleus/client/MailchimpClient.java

```diff
@@ -115,9 +115,17 @@ public List<MemberInfo> getListMembers(String listId) throws IOException, Mailch
   }
 
   public List<MemberInfo> getListMembers(String listId, String status, Calendar sinceLastChanged) throws IOException, MailchimpException {
+    return getListMembers(listId, status, null, sinceLastChanged);
```

Comment:
Pass in `"members.email_address,members.tags,total_items"` here instead of `null`?